.github/workflows/build-and-publish.yaml

```diff
@@ -28,10 +28,27 @@ jobs:
       - run: rush publish --include-all -p --force
         env:
           NODE_AUTH_TOKEN: ${{ secrets.NODE_AUTH_TOKEN }}
+
       - name: Commit changes
         uses: EndBug/add-and-commit@v9
         with:
           author_name: "Mozilla Rally Github CI/CD"
           author_email: "build@mozilla.com"
           message: "Bump versions [skip ci]"
           add: "*"
+
+      - uses: marceloprado/has-changed-path@v1
+        id: changed-functions
+        with:
+          paths: web-platform/functions
+
+      - name: Deploy firebase functions to production
+        if: steps.changed-functions.outputs.changed == 'true'
+        run: |
+            cp web-platform/firebase/firebase.config.rally-web-prod.json web-platform/firebase.config.json
+            cp web-platform/firebase/firestore.rules web-platform/firestore.rules
+            cp web-platform/firebase/firebase.json web-platform/firebase.json
```

Comment:
Instead of copying this and having to do the `sed` below, this could be used with `firebase --config`

Reply:
Unit tests within functions also use the same config and expects the functions to be at ../functions. Since we copy the file for deployment to the parent path we need to set this to functions.

Reply:
If I just use --config it complains that code is outside of the config folder.